templates/DatasetMonitoring/v2/researchGroups.html.twig

```diff
@@ -28,11 +28,11 @@
         columns: [{
             dataField: 'udi',
             caption: 'UDI',
-            width: '10rem',
+            width: '10.25rem',
             },{
                 dataField: 'doi',
                 caption: 'DOI',
-                width: '12rem',
+                width: '10.25rem',
```

Comment:
In addition could set `white-space:nowrap` to prevent hyphen from wrapping.
Also maybe try: `columnAutoWidth`

Reply:
columnAutoWidth worked perfectly. text-overflow: clip is destructive and makes the rest of a long string inaccessible (and non-displayed). With autowidth working right, no text was spilled ;-)